pkg/controller/plan/kubevirt.go

```diff
@@ -483,9 +483,9 @@ func (r *KubeVirt) EnsureVM(vm *plan.VMStatus) error {
 	}
 
 	for _, pvc := range pvcs {
-		ownerRefs := []meta.OwnerReference{vmOwnerReference(virtualMachine)}
 		pvcCopy := pvc.DeepCopy()
-		pvc.OwnerReferences = ownerRefs
+		pvc.OwnerReferences = createOwnerReferences(virtualMachine.ObjectMeta, metav1.TypeMeta{APIVersion: "kubevirt.io/v1",
```

Comment:
This is for the VM creation, thought that we agreed on this being the `BlockOwnerDeletion: true`